android/src/main/java/com/optimove/reactnative/events/PushOpenedEvent.java

```diff
@@ -44,7 +47,11 @@ public WritableMap getData() {
     map.putInt("id", push.getId());
```

Comment:
This mapping is duplicated between the two event classes, is there any value in making the logic shared so we reduce surface for future bugs/changes?

Reply:
Introduced an abstract PushEvent